ETC.tsv

```diff
@@ -2449,15 +2449,15 @@ ETC_20150317_002448	Obtained the herb!
 ETC_20150317_002449	Device seems to be open
 ETC_20150317_002450	Use a key.
 ETC_20150317_002451	Just leave it
-ETC_20150317_002452	Opend the device to use a key.
-ETC_20150317_002453	Say I know the place what you want to find.
-ETC_20150317_002454	Say I know the place what the stuff hidded by human.
-ETC_20150317_002455	Say that it can be a chance to impress Naktis
-ETC_20150317_002456	Say We have to hurry to find.
+xETC_20150317_002452	Open the device to use the key.
+xETC_20150317_002453	Say I know the place what you want to find.
+xETC_20150317_002454	Say I know the place what the stuff hidden by human.
+xETC_20150317_002455	Say that it can be a chance to impress Naktis
+xETC_20150317_002456	Say We have to hurry to find.
 ETC_20150317_002457	Now is the time.
 ETC_20150317_002458	Successfully persuaded.
 ETC_20150317_002459	I'll be there first!
-ETC_20150317_002460	Success transformation{nl}Leak the information to Naktis's suborinates.
+xETC_20150317_002460	Success transformation{nl}Leak the information to Naktis's subordinates.
```

Comment:
I know this isn't your edit, but:

Success transformation > Transformation success!
((including the exclamation mark)(